provisioning/kubernetes/templates/templates-api.yaml

```diff
@@ -78,6 +78,16 @@ spec:
                 - name: Host
                   value: KubernetesReadinessProbe
             periodSeconds: 10
+            failureThreshold: 1
+          livenessProbe:
+            httpGet:
+              path: /health-check
+              port: 8000
+              httpHeaders:
+                - name: Host
+                  value: KubernetesLivenessProbe
+            periodSeconds: 10
+            failureThreshold: 3
```

Comment:
Mame to teraz takto, pridal som `livenessProbe`:
https://github.com/keboola/keboola-as-code/blob/db153257052d6a79b7f9862e27474f70982cb995/provisioning/kubernetes/templates/templates-api.yaml#L64-L90

Reply:
Tj. `startupProbe`, sa spusta `30x1s`, a caka kym bude API ready pri starte.
`readinessProbe` sa spusta `1x10s` a hned ako zisti, ze API je nedostupne tak sa dany pod odpoji od requestov.
`livenessProbe` je nastavena na `3x10s`, tj. ak sa pod do `30s` neopravi, tak sa vypne.

Celkom to nesedi s nasou diskusiou:
- risili sme ze sa to deje pri deploy operacii
- a ze `readinessProbe` sposobuje restart, ale  `There's no way to trigger pod restart within a readiness probe. ` (https://stackoverflow.com/a/55260522).
- teda asi to neboli upne presne informacie, alebo neviem :) 
- doteraz sme tam `livenessProbe` nemali, tak by sa automaticky nemalo nic restartovat
- myslim si, ze tento config je teraz spravne vylepseny, ... chceme restart, ak `30s` pod neodpoveda, ... a ak pride nejaky DD alert, tak prosim sprav nejake podrobne screenshosts, ze to mozeme zanalyzovat.